docs/Reference.md

```diff
@@ -112,13 +112,14 @@ Clients may append more information and metadata to the _end_ of this string as
 
 The HTTP API implements a process for limiting and preventing excessive requests in accordance with [RFC 6585](https://tools.ietf.org/html/rfc6585#section-4). API users that regularly hit and ignore rate limits will have their API keys revoked, and be blocked from the platform. For more information on rate limiting of requests, please see the [Rate Limits](#DOCS_RATE_LIMITS/rate-limits) section.
 
->warn
->A bot account must connect and identify to a [Gateway](#DOCS_GATEWAY/connecting) at least once before being able to send messages.
-
 ## Gateway (WebSocket) API
 
 Discord's Gateway API is used for maintaining persistent, stateful websocket connections between your client and our servers. These connections are used for sending and receiving real-time events your client can use to track and update local state. The Gateway API uses secure websocket connections as specified in [RFC 6455](https://tools.ietf.org/html/rfc6455). For information on opening Gateway connections, please see the [Gateway API](#DOCS_GATEWAY/gateways) section.
 
+>warn
+>A bot must connect and identify to a gateway at least once before being able to use the [Create Message](#DOCS_CHANNEL/create-message) endpoint.
+>If you only want to send messages, consider using a [Webhook](#DOCS_WEBHOOK) instead.
```

Comment:
I like what this is getting at; maybe a little more specific:

`If you only want your bot to send messages and don't need to read them, consider using a Webhook instead.`

Reply:
Awkward phrasing since it implies that the user (you), not the bot, would not need to read the messages.

`If you only want your bot to send messages and [you] don't need to read them, consider using a Webhook instead.`
But the point is there.

How about:
`Users that wish to make an application that only sends messages and does not receive messages or other gateway events should consider using a webhook.`